ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -49,6 +49,7 @@ component:
   inverter: "Device or circuitry that changes direct current to alternating current (e.g. variable frequency drive)"
   lamp: "The component of a lighting device that emits the light."
   louver: "A device comprising multiple blades that regulates the flow of air"
+  luminaire: "Device that emits light"
```

Comment:
different from lamp above? @pisuke thoughts?

Reply:
I can also use "lamp". I thought that "luminaire" would include all parts of the device (lamp, dimmer, sockets etc.)

Reply:
I agree that it is useful to add luminaire too.
A better definition for "luminaire" would be:
"Complete lighting unit, consisting of one or more lamps, their socket/holders, wiring and optics such as lenses and reflectors to distribute the light"

Reply:
done